python/mlc_chat/model/mistral/mistral_loader.py

```diff
@@ -58,21 +58,81 @@ def huggingface(model_config: MistralConfig, quantization: Quantization) -> Exte
                 dtype=mlc_param.dtype,
             ),
         )
-        # Add gates in MLP
         mlp = f"model.layers.{i}.mlp"
         mlc_name = f"{mlp}.gate_up_proj.weight"
-        mlc_param = named_parameters[mlc_name]
-        mapping.add_mapping(
-            mlc_name,
-            [
-                f"{mlp}.gate_proj.weight",
-                f"{mlp}.up_proj.weight",
-            ],
-            functools.partial(
-                lambda gate, up, dtype: np.concatenate([gate, up], axis=0).astype(dtype),
-                dtype=mlc_param.dtype,
-            ),
-        )
+        if mlc_name in named_parameters:
```

Comment:
Let's use a separate folder for Mixtral given it's architecturally different than Mistral